From ea7f296345630c8351eea388c7fff6f3208f82dd Mon Sep 17 00:00:00 2001
From: Hao Dong <spdonghao@google.com>
Date: Fri, 11 Oct 2024 03:56:57 +0000
Subject: [PATCH 1/4] Set class name when createConfirmDeviceCredentialIntent()
 API is used.

Flag: EXEMPT bugfix
Bug: 339532378
Test: manual test
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:c608755d7606a6d301bf209e017289ffcbe76176)
Merged-In: I762f280def79d428ed9cf119ace5ccc2b6be2560
Change-Id: I762f280def79d428ed9cf119ace5ccc2b6be2560
---
 src/com/android/settings/password/BiometricFragment.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/settings/password/BiometricFragment.java b/src/com/android/settings/password/BiometricFragment.java
index e4c08e73ff8..04c82e559a3 100644
--- a/src/com/android/settings/password/BiometricFragment.java
+++ b/src/com/android/settings/password/BiometricFragment.java
@@ -140,6 +140,7 @@ public class BiometricFragment extends InstrumentedFragment {
                 .setDisallowBiometricsIfPolicyExists(
                         promptInfo.isDisallowBiometricsIfPolicyExists())
                 .setReceiveSystemEvents(true)
+                .setClassNameIfItIsConfirmDeviceCredentialActivity()
                 .build();
     }
 
-- 
2.40.1

